pool/tick_bitmap.gno

```diff
@@ -96,18 +85,30 @@ func (pool *Pool) tickBitmapNextInitializedTickWithInOneWord(
 	}
 }
 
-// bitwiseNotForUint256BitmapIndexing performs a bitwise NOT operation on the uint256 value.
-// This function is designed for use in bitmap indexing
-func bitwiseNotForUint256BitmapIndexing(x *u256.Uint) *u256.Uint {
-	// REF: https://stackoverflow.com/a/77071037
+func generateFullMaskForBitPos(bitPos uint) *u256.Uint {
+	// 1 << bitPos
```

Comment:
You can unify this `(1 << bitPos) - 1` comment. It's distracting.

Reply:
964aebbe02b53e6e0c716f58d20823933c188001